src/Sql/dbo/Stored Procedures/User_BumpAccountRevisionDateByCollectionId.sql

```diff
@@ -11,7 +11,7 @@ BEGIN
         U.[AccountRevisionDate] = GETUTCDATE()
     FROM
         [dbo].[User] U
-    LEFT JOIN
+    INNER JOIN
```

Comment:
Same, is this only to apply to org user (assuming so because collections are only used by organizations)?